packages/react/src/components/Tile/Tile.tsx

```diff
@@ -479,7 +479,7 @@ export const SelectableTile = React.forwardRef<
 
   function handleChange(event) {
     setIsSelected(event.target.checked);
-    onChange(event);
+    onChange(isSelected, name, event);
```

Comment:
Sometimes changing the signatures of callbacks like this can break folks relying on the exact signature. One thing to help avoid this would be to keep `event` in the same ( first) parameter position.

```suggestion
    onChange(event, isSelected, name);
```